plugins/convertPathData.js

```diff
@@ -844,7 +852,7 @@ function filters(
       }
 
       // remove useless non-first path segments
-      if (params.removeUseless && !maybeHasStrokeAndLinecap) {
+      if (params.removeUseless && isSafeToRemove(prev.command == 'M', true)) {
```

Comment:
I think `'m'` is also possible.

Reply:
Ah, right - while the first M is converted to absolute and excluded from converting to relative, the others aren't.